cmd/root.go

```diff
@@ -107,6 +115,9 @@ func NewShioriCmd(db dt.Database, dataDir string) *cobra.Command {
 	addCmd.Flags().StringSliceP("tags", "t", []string{}, "Comma-separated tags for this bookmark.")
 	addCmd.Flags().BoolP("offline", "o", false, "Save bookmark without fetching data from internet.")
 
+	checkCmd.Flags().BoolP("index-only", "i", false, "Only print the index of bookmarks")
+	checkCmd.Flags().BoolP("remove", "r", false, "Automatically remove non exiting bookmarks")
```

Comment:
`exiting` -> `existing`

Reply:
@techknowlogick thanks for pointing this out, pushed a new commit to adress this